system-variables.md

```diff
@@ -388,20 +388,12 @@ Constraint checking is always performed in place for pessimistic transactions (d
 
 ### tidb_enable_clustered_index <span class="version-mark">New in v5.0.0-rc</span>
 
-- Scope: SESSION | GLOBAL
-- Default value: OFF
-- This variable is used to control whether to enable the [clustered index](/clustered-indexes.md) feature.
-    - This feature is only applicable to newly created tables and does not affect the existing old tables.
-    - This feature is only applicable to tables whose primary key is the single-column non-integer type or the multi-column type. It does not affect the tables without a primary key or tables with the primary key of the single-column integer type.
-    - You can execute `select tidb_pk_type from information_schema.tables where table_name ='{table_name}'` to check whether the clustered index feature has been enabled on a table.
-- After you enable this feature, rows are stored directly on the primary key instead of on the internally allocated `rows_id` to which the extra primary key index is created to point.
-
-    This feature impacts performance in the following aspects:
-
-    - For each `INSERT` operation, there is one less index key written into each row.
-    - When you make a query using the primary key as the equivalent condition, one read request can be saved.
-    - When you make a query using the primary key as the range condition, multiple read requests can be saved.
-    - When you make a query using the prefix of the multi-column primary key as the equivalent condition or range condition, multiple read requests can be saved.
+- Scope: GLOBAL
+- Default value: INT_ONLY
+- This variable is used to control whether to create the primary key as a [clustered index](/clustered-indexes.md))by default, which is when the statement does not explicitly specifies the keyword `CLUSTERED`/`NONCLUSTERED`. Supported values are `OFF`, `ON`, and `INT_ONLY`:
+    - `OFF` indicates that primary keys are created as non-clustered indexes by default.
+    - `ON` indicates that primary keys are created as clustered indexes by default.
+    - `INT_ONLY` indicates that the behavior is controlled by the configuration item `alter-primary-key`. If `alter-primary-key` is set to `true`, primary keys are created as non-clustered indexes by default. If it is set to `false`, single integer primary keys are created as clustered indexes and others are created as non-clustered indexes.
```

Comment:
```suggestion
    - `INT_ONLY` indicates that the behavior is controlled by the configuration item `alter-primary-key`. If `alter-primary-key` is set to `true`, primary keys are created as non-clustered indexes by default. If it is set to `false`, single integer primary keys are created as clustered indexes and others primary keys are created as non-clustered indexes.
```

Reply:
If `alter-primary-key` is set to `true`, **all** primary keys are created as non-clustered indexes by default.

Reply:
```suggestion
    - `INT_ONLY` indicates that the behavior is controlled by the configuration item `alter-primary-key`. If `alter-primary-key` is set to `true`, all primary keys are created as non-clustered indexes by default. If it is set to `false`, single integer primary keys are created as clustered indexes and others are created as non-clustered indexes.
```